### PR TITLE
Fixed #28935 -- Fixed display of errors in extended blocks.

### PR DESCRIPTION
**Pull request by cammil**
Fixed #27694 -- Exception debug info on correct template.

### CODE REVIEW

**Comment by jacobtylerwalls** [comment]
Hi @cammil can you update this PR with the correct ticket number? Thanks. And please make sure the ticket links to the PR also.

**Comment by cammil** [comment]
@jacobtylerwalls Sorry for the wrong ticket number. Not sure how that happened! I've updated title with the correct number.

File: tests/template_tests/tests.py [hunk 1]
```diff
@@ -134,6 +135,26 @@ def test_compile_tag_error_27956(self):
...
+        """Test exception has correct debug info."""
+        e = Engine(
+            loaders=[(
+                'django.template.loaders.locmem.Loader', {
+                    'child': (
+                        '{% extends "parent" %}'
+                        '{% block content %}{% include "does_not_exist" %}{% endblock %}'
+                    ),
+                    'parent': '{% block content %}{% endblock %}',
+                })],
```
Comment (felixxm) [inline tests/template_tests/tests.py:148]: I moved `child` and `parent` to a separate templates.

**Review by felixxm — COMMENTED** [review]
@cammil Thanks for this patch :+1:

I pushed small edits.

**Comment by cammil** [comment]
@MalikRumi If you look at the PR the actual code change is pretty straightforward, so you could just modify your installed django.

**Pull request merged**